front50-gcs/front50-gcs.gradle

```diff
@@ -21,16 +21,10 @@ dependencies {
   implementation project(":front50-api")
 
   implementation "com.fasterxml.jackson.core:jackson-databind"
-  implementation ("com.google.apis:google-api-services-storage:v1-rev20200326-1.30.9") {
-       force=true
-  }
-  implementation ("com.google.auth:google-auth-library-oauth2-http:0.20.0") {
-       force=true
-  }
+  implementation "com.google.apis:google-api-services-storage"
+  implementation "com.google.auth:google-auth-library-oauth2-http"
   // TODO(plumpy): remove version once added to kork
```

Comment:
nit: this comment belongs next to the resolutionStrategy now.

Reply:
Update the comment.